course/src/main/java/org/openedx/course/presentation/container/CourseContainerViewModel.kt

```diff
@@ -353,6 +357,25 @@ class CourseContainerViewModel(
         _showProgress.value = false
     }
 
+    private fun updateContainerTabs(isDiscussionEnabled: Boolean) {
+        courseContainerTabs.value = CourseContainerTab.entries.filter {
+            it != CourseContainerTab.DISCUSSIONS || isDiscussionEnabled
+        }
+    }
+
+    fun getTabIndexByName(tabName: String): Int {
+        return courseContainerTabs.value.indexOfFirst {
+            it.name.equals(
+                tabName,
+                ignoreCase = true
+            )
+        }.takeIf { it != -1 } ?: 0
+    }
+
+    fun getTabNameByIndex(index: Int): CourseContainerTab {
```

Comment:
As the method returns the Tab enum type, not the name, IMO `getTabByIndex` OR `tabAt` is more appropriate here.